lib/taste_tester/commands.rb

```diff
@@ -210,5 +211,187 @@ def self.upload
       logger.error(exception.backtrace.join("\n"))
       exit 1
     end
+
+    def self.impact
+      logger.warn('Determining roles which will be impacted by current changes')
+
+      # Get the repository based on the paths specified in config.rb
+      repo = BetweenMeals::Repo.get(
+        TasteTester::Config.repo_type,
+        TasteTester::Config.repo,
+        logger,
+      )
+      if repo && !repo.exists?
+        fail "Could not open repo from #{TasteTester::Config.repo}"
+      end
+
+      # Compute the set of changes using Between Meals or a custom hook
+      if TasteTester::Config.use_custom_changeset_hook
+        changeset = TasteTester::Hooks.custom_changeset(repo)
+      else
+        changeset = _find_changeset(repo)
+      end
+
+      # Use the changeset computed earlier to find modified roles
+      if TasteTester::Config.use_custom_impact_hook
+        impact_roles = TasteTester::Hooks.custom_impact(changeset)
+      else
+        impact_roles = _find_impact(changeset)
+      end
+
+      if impact_roles.empty?
+        logger.warn('No impacted roles were found.')
+      else
+        logger.warn('The following roles have modified dependencies. ' +
```

Comment:
You're using `warn` in a lot of places where it's not a warning; this should probably be `info`, or perhaps just using `puts` if this is the "expected output of the command"  as opposed to logging information.

Reply:
In GD, only warn+ was getting sent to output, don't remember if that's the same in TT or not...

Reply:
TT only outputs warn+ by default.  I was mostly following the example of the rest of the commands and prod_hooks, but I can drop some of these down to info or puts.